src/chat/chatter.py

```diff
@@ -295,11 +296,13 @@ def pressed(self, item):
                 action_inspect_in_mordor = QtGui.QAction("Send the Orcs", menu)
                 menu.addAction(action_inspect_in_mordor)
 
+                ROOT = Settings.get('mordor/host')
+
                 def send_the_orcs():
                     if self.id != -1:
-                        QtGui.QDesktopServices.openUrl(QUrl("http://mordor.faforever.com/users/{}".format(self.id)))
+                        QtGui.QDesktopServices.openUrl(QUrl("http://" + ROOT + "/users/{}".format(self.id)))
```

Comment:
All the format stuff makes sense. Your final comment, though... No, not obvious. I think I get what you mean, and how it works in this case though. Is using %s better than using {} and format? Then write a function which grabs the target from config and puts it at the start?